src/core/agent/services/identifierService.ts

```diff
@@ -359,6 +408,15 @@ class IdentifierService extends AgentService {
     delegateContact?: ConnectionShortDetails
   ): Promise<string | undefined> {
     const ourMetadata = await this.getMetadataById(ourIdentifier);
```

Comment:
The name, colors and theme should be copied from `ourIdentifier` and not passed as an argument anymore, this is an acceptance criteria